corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -691,41 +690,32 @@ function GeoPointEntry(question, options) {
         self.rawAnswer([]);
     };
 
-    self.gMapsCallback = function () {
-        self.geocoder = new google.maps.Geocoder();
-        self.map = new google.maps.Map($('#' + self.entryId)[0], {
-            mapTypeId: google.maps.MapTypeId.ROADMAP,
-            center: new google.maps.LatLng(self.DEFAULT.lat, self.DEFAULT.lon),
-            zoom: self.DEFAULT.zoom,
-        });
-        if (self.rawAnswer().length) {
-            self.map.setCenter(new google.maps.LatLng(self.rawAnswer()[0], self.rawAnswer()[1]));
-            self.map.setZoom(self.DEFAULT.anszoom);
-        }
-        google.maps.event.addListener(self.map, "center_changed", self.updateCenter.bind(self));
-        var marker = new google.maps.Marker({
-            map: self.map,
-        });
-        marker.bindTo('position', self.map, 'center');
+    self.loadMap = function () {
+        self.map = L.map(self.entryId).setView([self.DEFAULT.lat, self.DEFAULT.lon], self.DEFAULT.zoom);
+        L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token='
+                    + window.MAPBOX_ACCESS_TOKEN, {
+            id: 'mapbox/streets-v11',
+        }).addTo(self.map);
+        self.map.on('move', self.updateCenter);
```

Comment:
More as an FYI for posterity, but I forgot to put the [attribution](https://docs.mapbox.com/help/how-mapbox-works/attribution/) to mapbox (and OpenStreetMaps) which is required.

Reply:
@ctsims verifying that this attribution to mapbox and OSM is okay. I believe we're using the Mapbox Streets Tileset which requires us to [attribute to both mapbox and OSM](https://docs.mapbox.com/help/how-mapbox-works/attribution/#text-attribution). 

I'm not too familiar with the other areas of leaflet, but it looks like we are not doin git in the report builder?
<img width="999" alt="Screen Shot 2020-07-09 at 5 16 09 PM" src="https://user-images.githubusercontent.com/615126/87091791-43213c80-c208-11ea-860b-4c68e5159870.png">

I tried to find it in the code and if this is it, it looks like the [attribution](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/static/reports/js/maps_utils.js#L348-L373) isn't passed properly? Do you want me to take a look?